packages/client/src/links/core.ts

```diff
@@ -45,7 +45,7 @@ export interface HTTPLinkOptions {
  */
 export type HttpLinkOptions = HTTPLinkOptions;
 
-export type HTTPHeaders = Record<string, string | string[] | undefined>;
+export type HTTPHeaders = Record<string, string | string[]>;
```

Comment:
This is a breaking change, so better if we filter them out

Reply:
Ok agreed! I have rolled back changes to the `HTTPHeaders` typing, `undefined`s will still be filtered out. 👍

Reply:
Also, setup github sponsors, I wna send you money 😉

Reply:
@KATT - I've just setup github sponsors 😊